.github/labeler.yml

```diff
@@ -3,12 +3,15 @@
 
 "Map-Admin":
  - "Resources/Maps/_NF/Admin/*.yml" # Grid Files
+ - "Resources/Maps/_NF/Admin/*/*.yml" # Grid Files
```

Comment:
Change to **?
```suggestion
 - "Resources/Maps/_NF/Admin/**/*.yml" # Grid Files
```

Reply:
If a file is Resources/Maps/_NF/Admin/test.ftl will it be marked as a map if i change this?